internal/payloads/reservation_payload.go

```diff
@@ -42,6 +42,10 @@ type GenericReservationResponsePayload struct {
 	Success *bool `json:"success" nullable:"true"`
 }
 
+type InstanceResponse struct {
+	*models.ReservationInstance
```

Comment:
We should probably do the same what we did in Amir's PR and for the Instances response (copy the instance details), but I guess that will be done while rebasing.

Reply:
rebased